project_charter.md

```diff
@@ -0,0 +1,103 @@
+# spack-stack Project Charter
+
+## Introduction
+
+spack-stack is a project which allows sysadmins and users to install
+correct versions of the many dependent packages that are required to
+build the Unified Forecast System (UFS) and other models.
+
+spack-stack relies on the open-source [spack
+project](https://github.com/spack/spack). Spack uses package
+configuration files to manage the building and dependencies of each
+package in the stack. (For example, the spack netcdf-c package file
+manages the installation of the netCDF C library.)
+
+All package configuration files used by spack-stack are maintained in
+the main spack repository, making them available for all
+users. spack-spack is a thin shell on spack which contains necessary
+configuration files for our HPC systems, and some utilities which aid
+in installing the large number of packages managed by spack-stack.
+
+## Stakeholders
+
+- NOAA Environmental Modeling Center (EMC)
+- The UCAR Joint Center for Satellite Data Assimilation (JCSDA)
```

Comment:
Any reason why there is a "The" in line 24 and in line 25, but not in 23?

Reply:
Fixed.